packages/store/src/StoreCore.sol

```diff
@@ -308,6 +309,8 @@ library StoreCoreInternal {
   bytes32 internal constant SLOT = keccak256("mud.store");
   bytes32 internal constant SCHEMA_TABLE = keccak256("mud.store.table.schema");
 
+  event MudStoreSetRecord(bytes32 table, bytes32[] key, bytes data);
+
   error StoreCore_InvalidDataLength(uint256 expected, uint256 received);
```

Comment:
I think you could use StoreCore's events/errors instead of declaring them again

Reply:
good point!